src/app/features/section/shared/interfaces/section-form.interface.ts

```diff
@@ -0,0 +1,11 @@
+export interface organizers {
+  firstName?: string;
+  lastName?: string;
+  id?: number;
+}
```

Comment:
skoro wiemy, ze będzie tylko 1 organizator, to zmienmy nazwę na Organizer (interfejsy piszmy tez z wielkiej litery);  

dlaczego wszystkie pola są opcjonalne?